fs/filter.go

```diff
@@ -385,6 +389,23 @@ func (f *Filter) includeRemote(remote string) bool {
 	return true
 }
 
+// ListContainsExcludeFile checks if exclude file is present in the list.
+func (f *Filter) ListContainsExcludeFile(entries DirEntries) bool {
+	if len(f.ExcludeFile) == 0 {
+		return false
+	}
+	for _, entry := range entries {
+		obj, ok := entry.(Object)
+		if ok {
+			_, leaf := RemoteSplit(obj.Remote())
```

Comment:
I think you want `path.Base(obj.Remote())` here - `RemoteSplit` is for processing the `remote:path` urls.  Not very well named or documented, I'd agree!